server/server/Controllers/adminCrud.js

```diff
@@ -0,0 +1,104 @@
+const itemsModel  = require("../models/Item");
+
+// Admin get item by id
+const getItemById = async (req, res) => {
+    const { id } = req.params;
+    try {
+        const item = await itemsModel.findById(id);
+        if (!item) {
+            // If the item is not found
+            return res.status(404).json({ message: "Item not found" });
+        }
+        // If the item is found
+        res.json(item);
+    } catch (err) {
+        // If there's an error
+        res.status(500).json({ message: err.message });
+    }
+};
+
+
+
+// admin add new item
+const addNewItem = async (req, res) => {
+  try {
+    const { title, price, itemPicture, category, availableQuantity } =
+      req.body;
+
+    const newItem = itemsModel({
+      title, 
+      price, 
+      itemPicture, 
+      category, 
+      availableQuantity,
+    });
```

Comment:
lets add a description field here, also add restaurantID field since it's required in schema

```suggestion
    const { title, description, restaurantId, price, itemPicture, category, availableQuantity } =
      req.body;

    const newItem = itemsModel({
      title, 
      description,
      restaurantID,
      price, 
      itemPicture, 
      category, 
      availableQuantity,
    });
```